libs/message-coder/src/bits.test.ts

```diff
@@ -0,0 +1,76 @@
+import { err, ok } from '@votingworks/types';
+import { Buffer } from 'buffer';
+import * as fc from 'fast-check';
+import {
+  bufferContainsBitOffset,
+  toBitLength,
+  toBitOffset,
+  toByteLength,
+  toByteOffset,
+} from './bits';
+import { CoderError } from './types';
+
+test('toByteLength', () => {
+  expect(toByteLength(0)).toEqual(0);
+  expect(toByteLength(1)).toEqual(1);
+  expect(toByteLength(4)).toEqual(1);
+  expect(toByteLength(7)).toEqual(1);
+  expect(toByteLength(8)).toEqual(1);
+  expect(toByteLength(9)).toEqual(2);
+
+  fc.assert(
+    fc.property(fc.integer(0, 0xffff), (value) => {
+      expect(toByteLength(value)).toEqual(Math.ceil(value / 8));
+    })
+  );
```

Comment:
What are the advantages of `fast-check` and why do you use it in this library?

Reply:
It generates a bunch of examples to check and, if it finds a counterexample, will do the work to reduce it to the simplest failing example it can. It helps check the edge cases that you might forget to try.